Dockerfile

```diff
@@ -0,0 +1,127 @@
+## Builds the hca-ingest dev env image
+
+# This is the Dagster user code deployment image
+# Dockerfile is located in orchestration/Dockerfile
+# FROM us.gcr.io/broad-dsp-gcr-public/monster-hca-dagster:latest
+FROM us.gcr.io/broad-dsp-gcr-public/monster-hca-dagster:WIP_bhill
```

Comment:
Will this eventually point at non-WIP tag?

Reply:
Ok that's all set now. Thank you for catching that!